src/hotspot/share/gc/shenandoah/shenandoahVerifier.cpp

```diff
@@ -305,7 +305,8 @@ class ShenandoahVerifyOopClosure : public BasicOopIterateClosure {
    */
   void verify_oops_from(oop obj) {
     _loc = obj;
-    obj->oop_iterate(this);
+    Klass* klass = obj->forward_safe_klass();
+    obj->oop_iterate_backwards(this, klass);
```

Comment:
Why `backwards`?

Reply:
Because that's the only oop_iterate() variant that takes a Klass* :-) It seems trivial to add such a forward-variant, though. Prefer that?